typing/ctype.ml

```diff
@@ -777,9 +777,9 @@ let rec copy_spine copy_scope ty =
           Tpoly (copy_rec ty', tvl)
       | Ttuple tyl ->
           Ttuple (List.map (fun (lbl, ty) -> (lbl, copy_rec ty)) tyl)
-      | Tpackage (path, fl) ->
+      | Tpackage {pack_path; pack_cstrs = fl} ->
           let fl = List.map (fun (n, ty) -> n, copy_rec ty) fl in
```

Comment:
I would propose to rename `fl` to ` pack_cstrs` to be more consistent on this line.

Reply:
Done